README.md

```diff
@@ -2,19 +2,23 @@
 
 http://mergely.com
 
-Mergely is a javascript component to diff/merge files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application.  It is suitable for comparing text files online, e.g. .txt, .html, .xml, .c, .cpp, .java, etc.  Mergely has a javascript implementation of the Longest Common Subsequence diff algorithm (LCS) and a customizable markup engine.
+Mergely is a javascript component to differencing and merging files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
+
+Mergely has a javascript implementation of the Longest Common Subsequence (LCS) diff algorithm, and a customizable markup engine.
 
 ## Installation
 
 ### Installation via webpack
-The recommended way to install mergely is to use npm and [webpack](https://webpack.js.org/) to install mergely and its dependencies.  It is highly recommended that you start by cloning [mergely-webpack](https://github.com/wickedest/mergely-webpack).  It has everything that you need to get started.
+The recommended way to install mergely and its dependencies is to use a Node package manager (npm) and [webpack](https://webpack.js.org/).
+
+It is highly recommended that you start by cloning [mergely-webpack](https://github.com/wickedest/mergely-webpack). The webpack has everything that you need to get started.
 
 ### Angular 6.1.1
-You can also use mergely within angular.  You can start by cloning [mergely-angular](https://github.com/wickedest/mergely-angular).
+You can also use mergely within angular. You can start by cloning [mergely-angular](https://github.com/wickedest/mergely-angular).
 
 ### Installation via .tgz
 
-Unpack mergely.tgz into a folder, e.g. `./lib`, and then add the following to the `<head>` of your target html source file.
+Unpack mergely.tgz into a folder, for example, `./lib`, and add the following to the `<head>` of your target HTML source file.
```

Comment:
```suggestion
Unpack mergely.tgz into a folder, for example `./lib`, and add the following to the `<head>` of your target HTML source file.
```

Reply:
https://www.thefreedictionary.com/for+example

Reply:
None of those examples apply to this case, IIUC. But that's a minor nit, really, so no need to bikeshed it :)